integrations/bitbucket-cloud/bitbucket_cloud/helpers/file_entity_handler.py

```diff
@@ -0,0 +1,67 @@
+import json
+from typing import Any, Dict, Type, Optional
+from loguru import logger
+from port_ocean.core.handlers import JQEntityProcessor
+from bitbucket_cloud.client import BitbucketClient
+import yaml
+
+
+FILE_PROPERTY_PREFIX = "file://"
+JSON_SUFFIX = ".json"
+YAML_SUFFIX = (".yaml", ".yml")
+
+
+class FileEntityProcessor(JQEntityProcessor):
+    prefix = FILE_PROPERTY_PREFIX
+
+    async def _get_file_content(
+        self, client: BitbucketClient, repo_slug: str, ref: str, file_path: str
+    ) -> Optional[Any]:
+        """Helper method to fetch and process file content."""
+        try:
+            file_content = await client.get_repository_files(repo_slug, ref, file_path)
+            if file_path.endswith(YAML_SUFFIX):
+                return yaml.safe_load(file_content)
+            elif file_path.endswith(JSON_SUFFIX):
+                return json.loads(file_content)
+            else:
+                return file_content
+        except Exception as e:
+            logger.error(f"Failed to get file content for {file_path}: {e}")
+            return None
+
+    async def _search(self, data: Dict[str, Any], pattern: str) -> Any:
+        """Search for a file in the repository and return its content."""
+        client = BitbucketClient.create_from_ocean_config()
+        repo_data = data.get("repo", data)
```

Comment:
maybe worth having a global bitbucket client that we would like to use instead of initializing it here and then passing it to the `_get_file_content`